### PR TITLE
DOM-22443: Initial verison of policies/terraform/readme

### PR DESCRIPTION
**Pull request by Secretions**
* Our restricted IAM policies
* Terraform module surrounding them
** Templetizes the stage name portion
** Can optionally provision an associated IAM account
** Can optionally provision (relatively unrestricted) Route53 access for internal dev purposes

### CODE REVIEW

File: iam-policy-a.json [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+        },
+        {
+            "Sid": "acm2",
+            "Effect": "Allow",
+            "Action": [
+                "acm:ListCertificates",
+                "acm:ListTagsForCertificate"
+            ],
+            "Resource": "*"
+        },
```
Comment (steved) [inline iam-policy-a.json:18]: Is this used anymore now that we do it in K8s?

File: variables.tf [hunk 2]
```diff
@@ -0,0 +1,5 @@
+variable "aws_region" { default = "" }
+variable "stage" { default = "domino" }
+variable "create_user" { default = false }
+variable "dev" { default = false } # Route53 policies for internal testing
+
```
Comment (steved) [inline variables.tf:5]: nitpick: ✂️  the whitespace

**Review by steved — APPROVED** [review]

File: iam-policy-a.json (hunk 1 above)
Comment (Secretions) [reply iam-policy-a.json:18]: Hm, this just made me realize a major flaw with this overall approach (that of the internal deployer PR, that is): It doesn't account for any aws calls our deployer makes. It's only testing the majority of aws calls that are performed by terraform, but not the ones the deployer itself will do.

**Comment by Secretions** [comment]
Going to merge this in as-is. It was semi-abandoned work that's going to be the base for further work by other people, and the repo is currently empty. Silly workflow to either add to months old PR by someone else or PR into old PR. Just merging so we can have the canon thing being modified for subsequent work.

**Pull request merged**